src/components/results/shortcircuit/shortcircuit-analysis-result-table.tsx

```diff
@@ -331,6 +335,68 @@ const ShortCircuitAnalysisResultTable: FunctionComponent<
         }
     }, []);
 
+    const onFilterChanged = useCallback(
+        (e: FilterChangedEvent) => {
+            // to see what contains filter model : https://www.ag-grid.com/javascript-data-grid/filter-api/
+            const formattedFilter = Object.entries(e.api.getFilterModel())
```

Comment:
I understand this code, but it's a pity that we don't use the raw filterModel format from AGGrid I think. You flatten the filters as well but it remove filters composition rules no ? How do you know which order to follow then and all?

Reply:
I wanted to make it a bit different of the filterModel, which can take different JSON form depending how many filters are activated by columns. Here you just send an array of filters, and the back manage it.
And yeah the drawback of this was no OR conditions was possible yet...
Anyway we won't use filterModel soon as discussed, and this will probably change...

Reply:
Actually as we decided to use only one filter by column now this code is obsolete (and will be remove soon anyway), so I will just simplify this part.